.github/workflows/java-ci.yml

```diff
@@ -37,11 +37,28 @@ jobs:
         path: ~/.gradle/caches
         key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
         restore-keys: ${{ runner.os }}-gradle
-    - run: ./gradlew check
+    - run: ./gradlew check -Pquick=true -x javadoc
     - uses: actions/upload-artifact@v2
       if: failure()
       with:
         name: test logs
         path: |
           **/build/testlogs
 
+  extra-checks:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
```

Comment:
Are we going to do these checks with java 8 only?

Reply:
Yes. I think that errorprone and checkstyle should be the same across both.

Reply:
Alright with me.